Generator/doc/Generator/CGAL/point_generators_3.h

```diff
@@ -335,6 +335,213 @@ namespace CGAL {
 
 /*!
 
+The class `Random_points_on_triangle_mesh_3` is an input iterator creating points uniformly
+distributed inside the faces of a triangle mesh model of `FaceListGraph`.
+
+\cgalModels `InputIterator`
+\cgalModels `PointGenerator`
+
+\sa `CGAL::cpp11::copy_n()`
+\sa `CGAL::Counting_iterator`
+\sa `CGAL::Random_points_in_disc_2<Point_2, Creator>`
+\sa `CGAL::Random_points_in_cube_3<Point_3, Creator>`
+\sa `CGAL::Random_points_in_triangle_3<Point_3, Creator>`
+\sa `CGAL::Random_points_on_sphere_3<Point_3, Creator>`
+\sa `CGAL::Random_points_in_triangle_mesh_2<Point_2, Triangulation>`
+\sa `CGAL::Random_points_on_tetrahedral_mesh_boundary<C3t3>`
+\sa `CGAL::Random_points_in_tetrahedral_mesh_3<C3t3>`
+\sa `std::random_shuffle`
+
+*/
+template <class Point_3, class TriangleMesh>
+class Random_points_on_triangle_mesh_3 {
+public:
```

Comment:
`boost::property_traits<VertexPointMap>::value_type` must be `Point_3` (note that Point_3 might be needed as template parameter since it can be deduced from `VertexPointMap`)